release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1366,6 +1387,16 @@ Now that setting `crun` as the container runtime is generally available when a p
 
 * Previously, the `openshift-manila-csi-driver` namespace did not include labels that are required for the management of workload partitioning. These missing labels impacted the operation of restricting Manila CSI pods to run on a selected set of CPUs. With this update, the `openshift-manila-csi-driver` namespace now includes the `workload.openshift.io/allowed` label. (link:https://issues.redhat.com/browse/OCPBUGS-11341[*OCPBUGS-11341*])
 
+[discrete]
+[id="ocp-4-13-windows-containers-bug-fixes"]
+==== Windows containers
+
+* Previously, Microsoft Windows container workloads were not completely emptied during the Windows node upgrade process. This resulted in service disruptions because the workloads remained on the node being upgraded. With this update, the Windows Machine Config Operator (WMCO) draines workloads and then cordones nodes until node upgrades finish. This action ensures a seamless upgrade for Microsoft Windows instances. (link:https://issues.redhat.com/browse/OCPBUGS-5732[*OCPBUGS-5732*])
+
+* Previously, the the Windows Machine Config Operator (WMCO) could not drain `DaemonSet` workloads. This issue caused Windows`DaemonSet` pods to block Windows nodes that the WMCO attempted to remove or upgrade. With this update, an WMCO includes additional role-based access control (RBAC) permisions, so that the WMCO can remove `DaemonSet` workloads. An WMCO can also delete any processes that were created with the `containerd` shim, so that `DaemonSet` containers do not exist on a Windows instance after a WMCO removes a node from a cluster. (link:https://issues.redhat.com/browse/OCPBUGS-5354[*OCPBUGS-5354*])
```

Comment:
```suggestion
* Previously, the Windows Machine Config Operator (WMCO) could not drain `DaemonSet` workloads. This issue caused Windows`DaemonSet` pods to block Windows nodes that the WMCO attempted to remove or upgrade. With this update, an WMCO includes additional role-based access control (RBAC) permisions, so that the WMCO can remove `DaemonSet` workloads. An WMCO can also delete any processes that were created with the `containerd` shim, so that `DaemonSet` containers do not exist on a Windows instance after a WMCO removes a node from a cluster. (link:https://issues.redhat.com/browse/OCPBUGS-5354[*OCPBUGS-5354*])
```

Reply:
Thanks, @snarayan-redhat . Updated.